03/app.js

```diff
@@ -1,3 +1,9 @@
 export default function randomNumber(min, max) {
+    if (typeof min !== 'number') {
```

Comment:
Przydałoby się jeszcze sprawdzić czy `max` to też liczba oraz czy `min` nie jest większe od `max`